config/kme_config.toml

```diff
@@ -81,8 +81,9 @@ ReceiptInvocationAddress = ""
 # Ethereum Address used by the worker to submit transactions
 FromAddress = ""
 # Replace ApplicationTypeId and OrganizationId with real hexadecimal values
-ApplicationTypeId = "11aa22bb33cc44dd"
-OrganizationId = "aabbcc1234ddeeff"
+ApplicationTypeId = "11aa22bb33cc44dd11aa22bb33cc44dd11aa22bb33cc44dd11aa22bb33cc44dd"
```

Comment:
Is ApplicationTypeId need to be hex string of length 64 chars? If so, can you add a comment?
Same applies to OrganizationId as well.